app/views/buyers/accounts/show.html.slim

```diff
@@ -62,13 +62,13 @@ h1 data-hook="account-show"
                                  reject_admin_buyers_account_path(@account),
                                  method: :post,
                                  class: 'reject action'
-            - if can? :suspend, @account
+            - if !@account.suspended? && !@account.scheduled_for_deletion?
```

Comment:
I wonder is this business rule should be a method of the decorator (if it's only useful in the UI) or the model (since it seems to be useful [elsewhere](https://github.com/3scale/porta/blob/402c4f2af6d4ff8a03f5f946758e23cb6b65109e/app/workers/process_notification_event_worker.rb#L40) too).